wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_main-nav.scss

```diff
@@ -268,8 +268,8 @@ body.explorer-open {
     }
 
     .nav-main {
-        position: absolute;
-        top: 175px; // WARNING - magic number - the height of the logo plus search box
+        position: relative;
+        top: 0;
         bottom: 0;
         margin-bottom: 127px; // WARNING - magic number - the height of the .footer
         width: 100%;
```

Comment:
Setting positions to 0 with `position: relative;` does nothing, so I have a feeling all of this code is useless unless it overrides mobile-specific rules.